Project cleanup

emilie-martin opened the pull request:
Big project cleanup! closes #42 
* uniformized the indentation, import order and general code formatting
* separated pages from components
* transitioned most of the `.css` files to `.ts` files to use `useStyles` with Material UI
* broke `orderBike` into components:
    * `BikeBilling` takes care of the billing
    * `BikeComponents` takes care of displaying the bike components to be selected by the user
    * `BikeModel` takes care of displayed the selected bike parts
 > the same logic breakdown was applied to for orderComponent

Functionality remains untouched and tests all pass.

Manraj45 reviewed (changes requested):
The "i" should be capital. In your app.tsx and UserLogs.tsx, your import has a capital while your file is not.
![image](https://user-images.githubusercontent.com/48056716/112227856-af14cb00-8c06-11eb-8b02-56f21b59e3a1.png)
![image](https://user-images.githubusercontent.com/48056716/112227992-e7b4a480-8c06-11eb-8a00-777fc1792577.png)

Manraj45 reviewed (changes requested):
Registration is broken. @nasaku898 had this issue and fixed it by removing the apostrophe from the recovery question. For example: What is your pet's name? should be: What is your pet name?
![image](https://user-images.githubusercontent.com/48056716/112228572-e6d04280-8c07-11eb-8197-76075923bb84.png)

emilie-martin commented:
> The "i" should be capital. In your app.tsx and UserLogs.tsx, your import has a capital while your file is not.
> ![image](https://user-images.githubusercontent.com/48056716/112227856-af14cb00-8c06-11eb-8b02-56f21b59e3a1.png)
> ![image](https://user-images.githubusercontent.com/48056716/112227992-e7b4a480-8c06-11eb-8a00-777fc1792577.png)

While I was making those changes I noticed that error pop up but I was fairly sure I had fixed it because it all runs fine on my end? Should I just change it manually from Git? Because it looks fine on my end and the UserLogs component works fine on my end.
![image](https://user-images.githubusercontent.com/44104876/112228850-51817e00-8c08-11eb-880b-c470b40a2f07.png)

Also, I'm not too sure why we're trying to get the `InventoryStyles` . I'll change it to ` UserLogsStyles`

emilie-martin commented:
> Registration is broken. @nasaku898 had this issue and fixed it by removing the apostrophe from the recovery question. For example: What is your pet's name? should be: What is your pet name?
> ![image](https://user-images.githubusercontent.com/48056716/112228572-e6d04280-8c07-11eb-8197-76075923bb84.png)

Oh I see. I had changed it so that it was proper English but I'll revert it. Must have forgotten to double-check the registration after changing it; that was my bad.

Manraj45 commented:
> Also, I'm not too sure why we're trying to get the `InventoryStyles` . I'll change it to ` UserLogsStyles`

That was @daniela-venuta  and @davidthai0387 part just make sure it's fine with them because maybe there was a reason behind it.

nasaku898 commented:
> While I was making those changes I noticed that error pop up but I was fairly sure I had fixed it because it all runs fine on my end? Should I just change it manually from Git? Because it looks fine on my end and the UserLogs component works fine on my end.
> ![image](https://user-images.githubusercontent.com/44104876/112228850-51817e00-8c08-11eb-880b-c470b40a2f07.png)
> 

I think Git doesn't pick up file name change since the name is still the name. I'd recommend just modify something in that file and commit it

Manraj45 commented on bike-erp/src/components/OrderService/OrderComponent/ComponentBilling.tsx:
```diff
@@ -1,21 +1,23 @@
-import { Order, removeAllItem, removeItem } from "../../redux/actions/OrderListActions/orderListAction";
+import { useState, useEffect } from "react";
 import { connect } from "react-redux";
+
```
Remove Space

emilie-martin replied:
As agreed offline, we'll keep the space to separate the import blocks and to adhere to the coding conventions 👍

Manraj45 commented on bike-erp/src/components/OrderService/OrderComponent/ComponentBilling.tsx:
```diff
@@ -1,21 +1,23 @@
+
+import { BACKEND_URL } from "../../../core/utils/config";
+import { Order, removeAllItem, removeItem } from "../../../redux/actions/OrderListActions/orderListAction";
+
```
Remove Space

emilie-martin replied:
As agreed offline, we'll keep the space to separate the import blocks and to adhere to the coding conventions 👍

Manraj45 commented on bike-erp/src/components/OrderService/OrderComponent/ComponentView.tsx:
```diff
@@ -1,43 +1,30 @@
 import bike_logo from "../../../assets/images/login_bike_logo.png";
-import { Box,Typography } from "@material-ui/core";
-import "./ModelView.css";
 
```
Remove Space

emilie-martin replied:
As agreed offline, we'll keep the space to separate the import blocks and to adhere to the coding conventions 👍

Manraj45 commented on bike-erp/src/components/PermissionManagement/PermissionDropdown.tsx:
```diff
@@ -1,9 +1,10 @@
+import { useState } from "react";
+
```
Remove space plz

emilie-martin replied:
As agreed offline, we'll keep the space to separate the import blocks and to adhere to the coding conventions 👍

Manraj45 commented on bike-erp/src/components/PermissionManagement/UserAccountsTable.tsx:
```diff
@@ -1,18 +1,24 @@
-import SaveIcon from "@material-ui/icons/Save";
 import axios from "axios";
 import { useEffect, useState } from "react";
+
```
Remove space here too

emilie-martin replied:
As agreed offline, we'll keep the space to separate the import blocks and to adhere to the coding conventions 👍

Manraj45 commented on bike-erp/src/index.tsx:
```diff
@@ -1,13 +1,14 @@
-import React from "react";
 import ReactDOM from "react-dom";
 import { Provider } from "react-redux";
+
```
Remove Space here

emilie-martin replied:
As agreed offline, we'll keep the space to separate the import blocks and to adhere to the coding conventions 👍

nasaku898 commented on bike-erp/src/test/Inventory/Inventory.test.tsx:
```diff
@@ -1,8 +1,8 @@
 import Enzyme, { shallow } from "enzyme";
```
Could you remove the console.log on this test. I requested it last time, but it was not removed.

emilie-martin replied:
It has been done 👍 hadn't even noticed them. If I had, I would have removed them 👍

Manraj45 reviewed (approved):
Good job! Don't forget to squash merge
